src/config/config.json

```diff
@@ -17,6 +17,8 @@
 	"PEER_NODE_PORT": 7900,
 	"REQUEST_TIMEOUT": 5000,
 	"NUMBER_OF_NODE_REQUEST_CHUNK": 10,
+	"NODE_PEERS_REQUEST_CHUNK_SIZE": 50,
+	"CHAIN_HEIGHT_REQUEST_CHUNK_SIZE": 10,
```

Comment:
Do you think can we just create a global chunk size that will work?

Reply:
I specifically avoided that, there wouldn't be just one chunk size that will fit all jobs. Managing it in different variables will allow us to adjust it individually in the runtime(via env vars).

Reply:
Good thinking, @yilmazbahadir.